Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/SubmissionSettings.kt

```diff
@@ -39,6 +39,11 @@ class SubmissionSettings @Inject constructor(
         defaultValue = false
     )
 
+    val hasViewedTestResult = prefs.createFlowPreference(
```

Comment:
This needs to be reset when tests are removed or the app is reset via settings. I did not see that in this PR.

Reply:
This happens in SubmissionSettings with the clear() function

Reply:
Ok, that's true

Reply:
What is missing is resetting is when the test is removed. That does not call clear() as other settings should remain.

Reply:
Good point. I will implement this.

Reply:
Yes,  I think removing a test should not call clear() but have a more explicit behavior